### ISSUE TITLE
About page > team people cards/links have no focus ring

### ISSUE THREAD

**Issue by madcampos** [techtankto/techtank-site#179]
The cards for each person on the about page have no visible focus.

<img width="489" height="317" alt="Screenshot of a person card from the about page" src="https://github.com/user-attachments/assets/9d8e8857-5248-420a-9604-765caff7b2ce" />

**Comment by Justin-Bento** [comment]
The browser skips these cards below when navigating the website with keyboard and mouse. Do we skip these compact cards and only and borders to clickable cards.


<img width="1624" height="1024" alt="Image" src="https://github.com/user-attachments/assets/fa7542df-dcd6-4b54-b24b-c35303157d43" />
<img width="1624" height="551" alt="Image" src="https://github.com/user-attachments/assets/1dd6cc22-4340-4c0a-822e-fcd758806d68" />

**Comment by 0916dhkim** [comment]
solved by
- https://github.com/techtankto/techtank-site/pull/219
